x-pack/test_serverless/functional/test_suites/security/ftr/discover/context_awareness/cell_renderer.ts

```diff
@@ -7,78 +7,55 @@
 
 import kbnRison from '@kbn/rison';
 import expect from '@kbn/expect';
-import path from 'path';
 import { FtrProviderContext } from '../../../../../ftr_provider_context';
-import { SECURITY_ES_ARCHIVES_DIR } from '../../../constants';
 
 export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const PageObjects = getPageObjects(['common', 'timePicker', 'discover', 'svlCommonPage']);
   const testSubjects = getService('testSubjects');
-  const dataViews = getService('dataViews');
-  const esArchiver = getService('esArchiver');
   const queryBar = getService('queryBar');
 
-  describe('security root profile', () => {
+  describe('cell renderer', () => {
     before(async () => {
-      await PageObjects.svlCommonPage.loginAsViewer();
-      await esArchiver.loadIfNeeded(path.join(SECURITY_ES_ARCHIVES_DIR, 'auditbeat_single'));
+      await PageObjects.svlCommonPage.loginAsAdmin();
```

Comment:
Does this functionality require `Admin` privileges? Won't it work with `platform_engineer`?

Reply:
Thank you. It is working with `platform_engineer`. I have now fixed it in 8293ab06bd06cf95e3441e2f6f997b6866e0b9c8